server/src/main/java/org/apache/uniffle/server/ShuffleDataFlushEvent.java

```diff
@@ -90,6 +92,14 @@ public ShuffleBuffer getShuffleBuffer() {
     return shuffleBuffer;
   }
 
+  public StorageManager getStorageManager() {
```

Comment:
It's a little weird that ShuffleDataFlushEvent have a member field `StorageManager`.

Reply:
> It's a little weird that ShuffleDataFlushEvent have a member field `StorageManager`.

`selectStorageManager` will be called again In `org.apache.uniffle.server.storage.MultiStorageManager#updateWriteMetrics`